packages/plugins/src/foundry/utils/index.ts

```diff
@@ -576,10 +546,13 @@ export const getConfigArtifactForContractName = (
  */
 export const isInitCodeMatch = (
   actualInitCodeWithArgs: string,
-  artifact: ContractArtifact
+  artifact: {
+    bytecode: string
+    linkReferences: LinkReferences
+    constructorFragment?: ethers.ConstructorFragment
+  }
```

Comment:
Changed this function to accept a minimal amount of artifact-related info instead of the entire artifact. The functionality remains the same.